src/Serilog.Sinks.Splunk/Sinks/Splunk/SplunkJsonFormatter.cs

```diff
@@ -95,6 +95,87 @@ public class SplunkJsonFormatter : ITextFormatter
             _suffix = suffixWriter.ToString();
         }
 
+        /// <summary>
+        /// Construct a <see cref="SplunkJsonFormatter"/>.
+        /// </summary>
+        /// <param name="formatProvider">Supplies culture-specific formatting information, or null.</param>
+        /// <param name="renderTemplate">If true, the template used will be rendered and written to the output as a property named MessageTemplate</param>
+        /// <param name="index">The Splunk index to log to</param>
+        /// <param name="source">The source of the event</param>
+        /// <param name="sourceType">The source type of the event</param>
+        /// <param name="host">The host of the event</param>
+        /// <param name="customFields">Object that describes extra splunk fields that should be indexed with event see: http://dev.splunk.com/view/event-collector/SP-CAAAFB6 </param>
+        public SplunkJsonFormatter(
+            bool renderTemplate,
+            IFormatProvider formatProvider,
+            string source,
+            string sourceType,
+            string host,
+            string index,
+            CustomFields customFields)
```

Comment:
> "Should CustomFields customFields be an optional param?..."

Yes
But have I not done so by have two constructors one with customfields and one without? Or am I missing something? Do you want me to solve it differently